README.md

```diff
@@ -1,2 +1,88 @@
 # business-analytics
 Tools to produce Riff Business analytic reports on RiffEdu and RiffPlatform deployments
+
+This repository uses a third-party package called [Metabase][] to visualize usage metrics for Mattermost. Two docker containers are run; one using the metabase image and another using the
+official mysql image. Commands are provided to restore sql database backups from Mattermost to the mysql container. 
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Database Backups
+
+Download sql backups from the Riff Learning backup location (ask for the URL if you don't have it), and place all .sql files in the folder called 'database-backups' in the root of this repo.
+
+### Prerequisites
+
+- [docker CE][docker-install] (Community Edition)
+- [docker-compose][docker-compose-install]
+- git
+- make
+
+
+## Running the Software
+The first step is to clone this repository. The following commands are defined in a Makefile, and can be run as follows:
+
+```
+make up
+```
+This command will create the mysql and metabase containers and show the logs for each
+
+```
+make restore
+```
+Once the `make up` command has completed (about 2-3 minutes), run this command. It will restore databases from all .sql files in the folder ./database-backups
+
+```
+make up-detach
+```
+This command will create the mysql and metabase containers in detached mode (no logs). This is useful once you have run the `make restore` command in a previous session, and are just creating the containers again.
+
+```
+make down
+```
+This command will stop and remove the mysql and metabase containers. The mysql volume containing the sql backups will 
+persist locally however, so only make up or make up-detach is needed to start the software again.
+
+#### Metabase will be run on port 3000
+
+### Using and Configuring Metabase
+
+If you are using a backup from our current deployment of edu-nexted, the database name in the sql backups 
+will be 'mattermost_test'. This is what Metabase is currently configured to look for, so it will run automatically with this
+configuration. If you are restoring an sql backup which contains databases other than 'mattermost_test', see the section 
+'Setting Up New Databases in Metabase' below.
+
+## View the Dashboard
+
+Once you navigate to port 3000, you will be shown the metabase homepage. I have pinned the Mattermost dashboard to the 
```

Comment:
'navigate to port 3000' really isn't clear. To be clear you should say something about go to `http://localhost:3000`.
I just did this and did **NOT** see the metabase homepage you describe. What I get is a metabase login page asking for an email address and a password.
I was wondering about the idea of checking the metabase database into the repository and this is making me think that the instructions should include (or maybe just refer to the metabase setup instructions).
A quick google found this discussion of import/export of questions and dashboards, which sounds like it might be a thing to keep in the repo than a pre-setup database. Worth exploring anyway, before having to add an email and password to these instructions.

Reply:
You are correct, I should perhaps to the documentation that you need to obtain the username and password from the dev team. From looking into it, it seems you can export questions, but not entire dashboards, which seems like it would take a long time. I feel like checking in the database and logging in is the best option.